src/main.rs

```diff
@@ -80,8 +80,16 @@ fn try_main() -> Result<(), Box<dyn std::error::Error>> {
         println!("{}", solver::solution_to_string(solution)?);
     }
 
-    if let SolveStatus::TimedOut { elapsed } = solve_result.status {
-        eprintln!("⚠️  Timed out after {:.1}s; only partial solutions returned", elapsed.as_secs_f64());
+    match solve_result.status {
+        SolveStatus::TimedOut { elapsed } => {
+            eprintln!("⚠️  Timed out after {:.1}s; only partial solutions returned", elapsed.as_secs_f64());
+        }
+        SolveStatus::FoundEnough => {
+            eprintln!("✓ Stopped after finding {} solutions", solve_result.solutions.len());
```

Comment:
maybe be even more explicit that this is a complete success (i.e., exactly what they asked for) (e.g., "✓ Stopped after finding requested number of solutions ({})")